rpi-update

```diff
@@ -42,11 +44,15 @@ FW_REV=${1:-""}
 FW_REVFILE="${FW_PATH}/.firmware_revision"
 [ "${RPI_UPDATE_UNSUPPORTED}" -ne 0 ] && echo -e "You appear to be trying to update firmware on an incompatible distribution. To force update, run the following:\nsudo -E RPI_UPDATE_UNSUPPORTED=0 rpi-update" && exit 1
 
+if [[ ${QUIET_MODE} -eq 1 ]]; then
+    CURL_COMMAND="curl -s"
```

Comment:
Presumably this should be `CURL_COMMAND="${CURL_COMMAND} -s"` to preserve any user-supplied `$CURL_COMMAND` from line 37 ?